apps/console/src/features/identity-providers/components/settings/attribute-management/attribute-mapping-list-item.tsx

```diff
@@ -121,121 +143,115 @@ export const AttributeMappingListItem: FunctionComponent<AttributeMappingListIte
             form.resetFieldState("localClaimId");
+            setMappedInputValue("");
+            setSelectedLocalAttributeInputValue("");
         }
     };
 
     return (
-        <FinalForm
+        <Form
             onSubmit={ onFormSub }
+            uncontrolledForm={ true }
             initialValues={ mapping && {
                 localClaimId: mapping?.claim?.id,
                 mappedValue: mapping?.mappedValue
-            } }
-            render={ ({ handleSubmit, values, initialValues }: FormRenderProps): ReactNode => (
-                <SemanticForm onSubmit={ handleSubmit }>
-                    <Grid>
-                        <Grid.Row columns={ editingMode ? 3 : 2 }>
-                            <Grid.Column width={ editingMode ? 7 : 8 }>
-                                <FinalField
-                                    name="mappedValue"
-                                    type="input"
-                                    component={ InputAdapter }
-                                    required
-                                    label={ !editingMode && "External IdP Attribute" }
-                                    width={ 16 }
-                                    initialValue={ initialValues?.mappedValue }
-                                    aria-label="External IdP Attribute Mapping Value"
-                                    placeholder="Enter external IdP attribute"
-                                />
-                            </Grid.Column>
-                            <Grid.Column width={ editingMode ? 7 : 8 }>
-                                <FinalField
-                                    search
-                                    clearable
-                                    name="localClaimId"
-                                    type="select"
-                                    component={ SelectAdapter }
-                                    required
-                                    width={ 16 }
-                                    options={ getListOfAvailableAttributes() }
-                                    label={ !editingMode && "Maps to" }
-                                    placeholder="Select mapping attribute"
-                                    aria-label="Local Claim Attribute"
-                                    noResultsMessage="Try another attribute search."
-                                />
+            } }>
+
+            <Grid>
+                <Grid.Row columns={ editingMode ? 3 : 2 }>
+                    <Grid.Column width={ editingMode ? 7 : 8 }>
+                        <Field.Input
+                            required
+                            name="mappedValue"
+                            inputType="identifier"
+                            maxLength={ 120 }
+                            minLength={ 1 }
+                            label={ !editingMode && "External IdP Attribute" }
+                            placeholder="Enter external IdP attribute"
```

Comment:
Shall we translate these strings. Check other places as well.
```suggestion
                            placeholder="Enter external IdP attribute"
```

Reply:
All the locale and i18n issues related to saml IdP will address with `a:5593`